plenum/server/node.py

```diff
@@ -1648,6 +1648,26 @@ def handleInvalidClientMsg(self, ex, wrappedMsg):
                 reqId = getattr(ex, f.REQ_ID.nm, None)
         self.send_nack_to_client((identifier, reqId), reason, frm)
         self.discard(wrappedMsg, friendly, logger.info, cliOutput=True)
+        self.specific_invalid_client_msg_handling(ex, wrappedMsg)
+
+    def specific_invalid_client_msg_handling(self, ex, wrappedMsg):
```

Comment:
Please use `_` prefix since this is a private method.

Reply:
ok